internal/cli/cli.go

```diff
@@ -224,7 +229,7 @@ func (c *cli) addTenant(ten tenant) error {
 	c.config.Tenants[ten.Name] = ten
 
 	if err := c.persistConfig(); err != nil {
-		return fmt.Errorf("persisting config: %w", err)
+		return fmt.Errorf("Unexpected error persisting config: %w", err)
```

Comment:
error messages starting with lowercase (and without ending puctuation) plays nicely when rendered to the end-user, a log entry, etc.

more details here: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

```suggestion
		return fmt.Errorf("unexpected error persisting config: %w", err)
```

Reply:
Thanks!